my-config/dom0-powertop.sls

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# vim: set syntax=yaml ts=2 sw=2 sts=2 et :
+
+dom0:
+  pkg.installed:
+    - name: powertop
+    - fromrepo: fedora
+powertop:
+  service.dead:
```

Comment:
`service.dead` will stop the process even with the `enable: True` option. I suppose you meant to use `service.running` instead.

Reply:
Powertop is not a service, but oneshoot. It is ran and then die. When launched how you suggest, the formula fails because the service is dead when checked.

Reply:
> Powertop is not a service, but oneshoot.

Indeed, I forgot this. Consequently, why not using `service.enabled`?

I suggest you to rewrite the state this way:

```
powertop:
  pkg.installed:
    - fromrepo: fedora
  service.enabled: []
```

Then, I will accept your changes.